src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -2563,7 +2562,7 @@ public enum RisingRivals implements LogicCardInfo {
             energyCost C
             attackRequirement {}
             onAttack {
-              delayed {
+              delayed PRIORITY:BEFORE_LAST, {
```

Comment:
Are you sure this works? PRIORITY: should not be a map key I'm pretty sure
```suggestion
              delayed BEFORE_LAST, {
```
should just work

Reply:
This does work.